tests/NHapi.Base.NUnit/SupportClassTests.cs

```diff
@@ -0,0 +1,42 @@
+namespace NHapi.Base.NUnit
+{
+    using global::NUnit.Framework;
+
+    using NHapi.Base;
+
+    [TestFixture]
+    public class SupportClassTests
```

Comment:
add a test for null/empty/whitespace input?

Reply:
will do

Reply:
In regards to the generic `CharsAreUnique` whitespace chars are still valid chars - but will check for null or empty.